src/aggregate/aggregate_exec.c

```diff
@@ -631,7 +631,8 @@ static void sendChunk_Resp3(AREQ *req, RedisModule_Reply *reply, size_t limit,
     } else if (rc == RS_RESULT_ERROR) {
       // Non-fatal error
       RedisModule_Reply_SimpleString(reply, QueryError_GetUserError(qctx->err));
-    } else if (QueryError_HasReachedMaxPrefixExpansionsWarning(qctx->err)) {
+    }
```

Comment:
Why do we still keep it in  `if else` ?

Reply:
the rc code can't get more than one value

Reply:
So it shouldn't be only an else to timeout right?